src/main/java/seedu/findvisor/logic/parser/DeleteTagCommandParser.java

```diff
@@ -0,0 +1,45 @@
+package seedu.findvisor.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.findvisor.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.findvisor.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.findvisor.commons.core.index.Index;
+import seedu.findvisor.logic.commands.DeleteTagCommand;
+import seedu.findvisor.logic.parser.exceptions.ParseException;
+import seedu.findvisor.model.tag.Tag;
+
+/**
+ * Parses input arguments and creates a new DeleteTagCommand object
+ */
+public class DeleteTagCommandParser implements Parser<DeleteTagCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the
+     * DeleteTagCommand
+     * and returns a DeleteTagCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public DeleteTagCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_TAG);
+
```

Comment:
For this portion, you might have to add a check to see if the tag prefix is present in the deletetag call.